dashboard/app/graphs.go

```diff
@@ -414,30 +420,34 @@ func createManagersGraph(c context.Context, ns string, selManagers, selMetrics [
 	return graph, nil
 }
 
-func extractMetric(stat *ManagerStats, metric string) float64 {
+func extractMetric(stat *ManagerStats, metric string) (float64, bool) {
 	switch metric {
 	case "MaxCorpus":
-		return float64(stat.MaxCorpus)
+		return float64(stat.MaxCorpus), true
 	case "MaxCover":
-		return float64(stat.MaxCover)
+		return float64(stat.MaxCover), true
 	case "MaxPCs":
-		return float64(stat.MaxPCs)
+		return float64(stat.MaxPCs), true
 	case "TotalFuzzingTime":
-		return float64(stat.TotalFuzzingTime)
+		return float64(stat.TotalFuzzingTime), true
 	case "TotalCrashes":
-		return float64(stat.TotalCrashes)
+		return float64(stat.TotalCrashes), true
 	case "CrashTypes":
-		return float64(stat.CrashTypes)
+		return float64(stat.CrashTypes), true
 	case "SuppressedCrashes":
-		return float64(stat.SuppressedCrashes)
+		return float64(stat.SuppressedCrashes), true
 	case "TotalExecs":
-		return float64(stat.TotalExecs)
+		return float64(stat.TotalExecs), true
 	case "ExecsPerSec":
 		timeSec := float64(stat.TotalFuzzingTime) / 1e9
 		if timeSec == 0 {
-			return 0
+			return 0, false
 		}
-		return float64(stat.TotalExecs) / timeSec
+		return float64(stat.TotalExecs) / timeSec, true
+	case "TriagedCoverage":
+		return float64(stat.TriagedCoverage), stat.TriagedCoverage != 0
```

Comment:
How much value does this null/non-null add? Could we just use 0 as is?
If it's useful, should we consider most other metrics as null if the value is 0? If we have 0 coverage or 0 executions, this also effectively means we don't have the value.
Then perhaps we could just compare value with 0 in the rendering template?

Reply:
> How much value does this null/non-null add? Could we just use 0 as is?

Null points are not displayed on the graph. If we use 0 instead of nulls, the graph will be jumping day to day crazily. We don't really the problem for other parameters because it's quite rare for our instances to work on some days and not work on other days. For the added two parameters it's different -- our instances are quite often restarted every 2-3 days, but we'll have the values set only on the days when they were restarted.

> If it's useful, should we consider most other metrics as null if the value is 0?

Yes, I think it could be generalized to some other metrics. But not for all, e.g. crash count or suppressed crashes count could probably be 0 on a slow, but still normally working instance. 

> Then perhaps we could just compare value with 0 in the rendering template?

No, since 0 is a valid value at least for some of the metrics, it's not valid to move the check to the function. But we can add more  `, val != 0` conditions to the return expressions in the function.

Reply:
Ack, let's keep as is or add "val != 0" for all metrics except crash counts.